frontend/src/Components/Router/routes.ts

```diff
@@ -6,7 +6,8 @@ export const ADMIN = '/admin';
 
 export const BOOKS = `${INVENTORY}/libros`;
 export const NEW_BOOK = `${BOOKS}/nuevo`;
-export const LIST_LOCAL_BOOKS = `${BOOKS}/lista-local`;
+export const LIST_LOCAL_BOOKS = `${BOOKS}/buscar-local`;
```

Comment:
Aquí sugiero que cambiemos a esto:
```suggestion
export const LIST_LOCAL_BOOKS = `${BOOKS}/buscar`;
```